src/backend/distributed/commands/create_distributed_table.c

```diff
@@ -1671,7 +1671,10 @@ EnsureRelationCanBeDistributed(Oid relationId, Var *distributionColumn,
 	{
 		EnsureRelationHasNoTriggers(relationId);
 	}
-
+	else
+	{
+		ErrorIfRelationHasUnsupportedTrigger(relationId);
+	}
```

Comment:
small nit, since we have a negative check above, would you mind swapping the branches around and do a positive check. This makes the code easier to read for people coming by in the future.